pkg/bpf/utils/pin.go

```diff
@@ -74,8 +74,8 @@ func UnpinMaps(value *reflect.Value) error {
 func SetMapPinType(spec *ebpf.CollectionSpec, pinType ebpf.PinType) {
 	for key, v := range spec.Maps {
 		// tail_call map dont support pinning when shared by different bpf types
-		if strings.HasPrefix(key, ".rodata") || key == ".bss" || key == "kmesh_tail_call_prog" ||
-			key == "kmesh_tail_call_ctx" {
+		if strings.HasPrefix(key, ".rodata") || key == ".bss" || key == "km_tailcallprog" ||
+			key == "km_tailcallctx" {
```

Comment:
can we define some constants to remind developers to update it when they rename the map?